doc/2/guides/introduction/1-what-is-kuzzle/index.md

```diff
@@ -133,6 +133,16 @@ These plugins allow you to use the functionalities of other services such as [Am
 
 The community is also able to develop and distribute its own plugins to enrich the ecosystem.
 
+### Expert Professional Support
+
+The Kuzzle backend and all our projects are developed by a team of French engineers based in Montpellier.  
```

Comment:
🇫🇷 
```suggestion
The Kuzzle backend and all our projects are developed by a team of engineers based in Montpellier.  
```

Reply:
We also need to specify the country, as Montpellier is not very well-known (and there is a Montpellier in England and in Canada)

```suggestion
The Kuzzle backend and all our projects are developed by a team of engineers based in Montpellier, France.  
```